forge-gui/res/cardsfolder/upcoming/portal_manipulator.txt

```diff
@@ -0,0 +1,11 @@
+Name:Portal Manipulator
+ManaCost:2 WU WU
+Types:Creature Human Wizard
+PT:3/2
+K:Flash
+T:Mode$ ChangesZone | Phase$ Declare Attackers | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChoosePlayer | TriggerDescription$ When CARDNAME enters the battlefield during the declare attackers step, choose target player and any number of target attacking creatures their opponents control. Those creatures are now attacking that player.
+SVar:TrigChoosePlayer:DB$ Pump | ValidTgts$ Player | RememberObjects$ Targeted | SubAbility$ DBChooseAttackers
+SVar:DBChooseAttackers:DB$ ChangeCombatants | ValidTgts$ Creature.attacking+!RememberedPlayerCtrl | TargetMin$ 0 | TargetMax$ MaxTargets | TgtPrompt$ Select any number of target attacking creatures controlled by target player's opponents | Attacking$ Player.IsRemembered | SubAbility$ DBCleanup
```

Comment:
this is a bit tricky because the "their opponents" is relative
so you shouldn't be able to target player A and then creatures of B if they're teammates
try `ValidTgts$ Creature.attacking | TargetsWithDefinedController$ Player.OpponentOf ParentTargeted` instead

Reply:
A fair point. 
Though I do need a replacement for the line `SVar:MaxTargets:Count$Valid Creature.attacking+!RememberedPlayerCtrl` so the script can determine the maximum targets allowed. As it is, replacing `ValidTgts$ Creature.attacking+!RememberedPlayerCtrl | TargetMin$ 0 | TargetMax$ MaxTargets` with what you recommend and removing the `SVar:MaxTargets` line gets me as far as the player selection prompt which I follow into no effect, rather than the creature selection prompt as one would hope. 
I tried a few iterations for both lines after combing the scripts for likely leads but, besides no dice on those attempts dealing with teams adequately, I discovered that, confoundingly, there's `Allies` (on Imperial Mask), `NonOpponent` (on Organ Harvest), and `Team` (on Azra Bladeseeker and other relevant Battlebond cards), all referring to ostensibly the same concept, players teaming up. 
I do feel having something more generalized than the existing `YourTeamCtrl`, maybe `RememberedPlayerTeamCtrl`, would be the ticket here.

Reply:
Done and tested: works as expected.